Mathlib/Analysis/SpecialFunctions/Log/PosLog.lean

```diff
@@ -23,18 +23,20 @@ namespace Real
 /-!
 ## Definition, Notation and Reformulations
 -/
+
 /-- Definition: the positive part of the logarithm. -/
-noncomputable def posLog : ℝ → ℝ := fun r ↦ max 0 (log r)
+@[pp_nodot] noncomputable def posLog : ℝ → ℝ := fun r ↦ max 0 (log r)
```

Comment:
And I think we *want* `x.posLog` when the notation is not active, so this isn't desirable either.